dts/bindings/serial/aesc,uart.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2025 aesc silicon
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: aesc UART - This binding gives a base representation of the aesc UART
+
+compatible: "aesc,uart"
```

Comment:
Is this peripheral a general peripheral from `aesc` or is it specific to the ElemRV-N?

Reply:
General peripheral but used in ElemRV.